packages/components/NEW-COMPONENT-CHECKLIST.md

```diff
@@ -76,9 +76,13 @@ The engineering checklist has six parts: creating the feature branch, component
 ### Component Creation
 
 - [ ] create new branch from main for the component (`git checkout -b USER/COMPONENT-NAME`)
+  - Note: There is no standardized branch naming convention.
+  - Suggestion: Use Jira ticket number together with a brief description. (Including the ticket number may allow us to automatically link branches to tickets and PRs in the future.)
+  - Example Branch Name: "hds-123-fix-typo".
```

Comment:
Let me know if this is helpful or if I should edit/remove. Or if this should be done as a separate PR.

Reply:
A few considerations:
- I do `melsumner/branch-name` because it makes it easier to identify my branches (both in the branch UI and in tools like Tower)
- We have a spot in the PR description to link to the issue itself 

Either way, also agree with @didoo that this should be in a separate PR 👍

Reply:
@MelSumner So would you like me to undo these changes and create a separate PR later?

Reply:
I'll just undo this change since it doesn't seem like it's useful currently. 

I would like to try to get the feature to link branches to tickets & PRs working eventually if we can though since I think it's very helpful to connect them automatically vs. having to add a link more manually.